src/masternodes/govvariables/attributes.cpp

```diff
@@ -1810,25 +1810,6 @@ Res ATTRIBUTES::Validate(const CCustomCSView &view) const {
                         if (GetValue(intervalPriceKey, CTokenCurrencyPair{}) == CTokenCurrencyPair{}) {
                             return DeFiErrors::GovVarValidateCurrencyPair();
                         }
-
-                        const CDataStructureV0 enabledKey{AttributeTypes::Param, ParamIDs::Feature,
-                                                          DFIPKeys::EVMEnabled};
-
-                        CrossBoundaryResult result;
-                        if (view.GetLastHeight() >= Params().GetConsensus().NextNetworkUpgradeHeight &&
-                            GetValue(enabledKey, false) &&
-                            evmQueueId &&
-                            !evm_try_is_dst20_deployed_or_queued(result, evmQueueId, token->name, token->symbol,
-                                                       tokenID.v)) {
-                            evm_try_create_dst20(result, evmQueueId, token->creationTx.GetHex(),
-                                                 token->name,
-                                                 token->symbol,
-                                                 tokenID.v);
-
-                            if (!result.ok) {
-                                return DeFiErrors::GovVarErrorCreatingDST20(result.reason.c_str());
-                            }
-                        }
```

Comment:
It seems like shifting the ffi call to create dst20 tokens from here into CreateToken in the custom view will mean that the creation of dst20 tokens are no longer done on genesis block, but only on create token or create poolpairs or create loan tokens.